app/views/mechanics/show.html.erb

```diff
@@ -0,0 +1,15 @@
+<h1>Mechanic: <%= @mechanic.name %></h1>
+<p>Years of Experience: <%= @mechanic.years_experience %></p>
+
+<h3>Mechanic Current Rides</h3>
+<% @rides.each do |ride|%>
+  <%= ride.name %></br>
+<% end %>
+
+<h3>Add Ride To <%= @mechanic.name %> Workload</h3>
+<%= form_with url: mechanic_ride_path(@mechanic), method: :patch, data: { turbo: false } do |form| %>
+  <%= form.label :ride_id, "Ride ID:" %>
+  <%= form.text_field :ride_id %>
+  
+  <%= form.submit "Add Ride" %>
+<% end %>
```

Comment:
Everything here is great but the form. Your form isn't quite implemented properly because the verb and path are incorrect. If we run a `rails routes`, this route would be something like `"/mechanic_rides/:id"` which is not the nested resource routing that we are expecting. We want the route to be something like `"/mechanics/:id/rides"`. 

If you are using the `resources` syntax in your `routes.rb` this would mean that you needed to nest `:rides` within `:mechanics` which it looks like you didn't do.

The form is also supposed to be a `POST` and not a `PATCH` since we are creating a new record in the DB.